README.md

```diff
@@ -1,37 +1,265 @@
 # django-simple-nav
 
+<!-- intro-start -->
 [![PyPI](https://img.shields.io/pypi/v/django-simple-nav)](https://pypi.org/project/django-simple-nav/)
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/django-simple-nav)
 ![Django Version](https://img.shields.io/badge/django-4.2%20%7C%205.0%20%7C%205.1-%2344B78B?labelColor=%23092E20)
 <!-- https://shields.io/badges -->
 <!-- django-4.2 | 5.0 | 5.1-#44B78B -->
 <!-- labelColor=%23092E20 -->
 
+`django-simple-nav` is a Python/Django application designed to simplify the integration of navigation and menu bars in your Django projects. With a straightforward API and customizable options, you can easily add and manage navigational elements in your web applications. It is designed to be simple to start with, but flexible enough to handle complex navigation structures while maintaining that same simplicity.
+
 ## Requirements
 
-- Python 3.9, 3.10, 3.11, 3.12, 3.13
+- Python 3.8, 3.9, 3.10, 3.11, 3.12, 3.13
```

Comment:
Did you have second thoughts on dumping 3.8? (I think it can be a soft drop where we don't go out of our way to prevent it working on 3.8)

Reply:
Yeah, I'm not going out of my way to prevent it working, but I'm publicly dropping support the minute a version goes on this list: https://devguide.python.org/versions/#unsupported-versions.

Reply:
Though now that I think through it, maybe Django should be the driver of that -- and 3.8 is still supported under 4.2 https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django.

I could go either way I guess, not sure what the right decision is.

Reply:
Hm, I'm going to leave it alone for now and stick to if Django supports it, we'll support it as long as we aren't using a feature of Python not available in it.

Reply:
Oh... I see what you were asking now 🤦‍♂️

No, the copier update wiped away the README and I reverted it to the one on `main` and missed that.

Reply:
![giphy](https://github.com/user-attachments/assets/81b8a5ef-9d21-4434-affc-8efdce7c36dc)

EH, in light of that, dropping 3.8 seems fine.. What a wild ride.